dep11/generator.py

```diff
@@ -274,6 +296,8 @@ def handle_error(e):
 
             log.info("Completed metadata extraction for suite %s/%s" % (suite_name, component))
 
+        langpacks.cleanup()
```

Comment:
I thought about making the new class a context manager so that it's automatically cleaned up on exit, but then I'd have to indent the whole large block and that seemed ugly. What do you think?